model/model.md

```diff
@@ -16,24 +16,24 @@ Note that all relationships are multiplicities unless specified.
-- A `Participant Agent` performs tasks such as publishing a catalog or engaging in an asset transfer. In order to accomplish these tasks, a participant agent may
+- A `Participant Agent` performs tasks such as publishing a catalog or engaging in a dataset transfer. In order to accomplish these tasks, a participant agent may
   use a _**verifiable presentation**_ generated from a _**credential**_ obtained from a third-party issuer. A participant agent may also use an _**ID token**_ issued by a
   third-party identity provider. Note that a participant agent is a logical construct and does not necessarily correspond to a single runtime process.
 - An `Identity Provider` is a trust anchor that generates `ID tokens` used to verify the identity of a `Participant Agent`. Multiple identity providers may operate in
   a dataspace. The types and semantics of ID tokens are not part of this specification. An identity provider may be a third-party or a participant itself (for example, in the case
   of decentralized identifiers).
-- A `Credential Issuer` issues _verifiable credentials_ used by participant agents to allow access to assets and verify usage control.
+- A `Credential Issuer` issues _verifiable credentials_ used by participant agents to allow access to datasets and verify usage control.
 
 The diagram below depicts the relationships between `ParticipantAgent` types:
 
 ![](./m.participant.entities.png)
 
 - A `CatalogService` is a `Participant Agent` that makes a [DCAT Catalog](https://www.w3.org/TR/vocab-dcat-3/#Class:Catalog) available to other participants.
-- A `Catalog` contains one or more `Asset Entries`, which are [DCAT Datasets](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset). A `Catalog` also contains **_at least one_**
-  [DCAT DataService](https://www.w3.org/TR/vocab-dcat-3/#Class:Data_Service) that references a `Connector` where assets may be obtained.
-- An `Asset Entry` has **one** `Offer`, which is an [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the asset.
-- A `Connector` is a `Participant Agent` that performs `Contract Negotiation` and `Asset Transfer` operations with another connector. An outcome of a `ContractNegotiation` may
-  be the production of an `Agreement`, which is an [ODRL Agreement](https://www.w3.org/TR/odrl-model/#policy-agreement) defining the _usage control policy_ agreed to for an asset.
+- A `Catalog` contains one or more `Datasets`, which are [DCAT Datasets](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset). A `Catalog` also contains **_at least one_**
+  [DCAT DataService](https://www.w3.org/TR/vocab-dcat-3/#Class:Data_Service) that references a `Connector` where datasets may be obtained.
+- An `Dataset` has **one** `Offer`, which is an [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the dataset.
```

Comment:
Is this correct? To my understanding a `dataset` would contain multiple `offers` which is what I can find in `hasPolicy`. That was at least to conclusion when I asked this last time:
https://github.com/International-Data-Spaces-Association/ids-specification/issues/76#issuecomment-1493778662
We also updated the examples accordingly here:
https://github.com/International-Data-Spaces-Association/ids-specification/pull/80/files

Reply:
I just did a quick test with 2 EDCs. When I create 2 different policies and 2 contract definitions (with the same asset...) it produces the expected catalog result with a list of `hasPolicy` containing exactly the 2 previously created policies.

To my understanding the sentence needs to be updated to:

```
An `Dataset` has **multiple** `Offers`, which are a [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the dataset.
```